packages/dashboard/src/helpers/addValuesToDashboardFilters.ts

```diff
@@ -39,7 +43,16 @@ export const addValuesToDashboardFilters = (filters: SharedFilter[], data: Recor
         const active: string[] = Array.isArray(filterCopy.active) ? filterCopy.active : [filterCopy.active]
         filterCopy.active = active.filter(val => defaultValues.includes(val))
       } else {
-        const defaultValue = filterCopy.values[0] || filterCopy.active
+        let hasResetLabel = false

```

Comment:
I think resetLabel isn't being set in this code. You could also consider simplifying to:

```.js
const resetLabelFilter = filters.find(fil => fil.resetLabel);
const defaultValue = resetLabelFilter ? resetLabelFilter.resetLabel : filterCopy.values[0] || filterCopy.active;
```